java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2673,6 +2675,57 @@ public T touch(@NonNull String[] keys) {
         return getThis();
     }
 
+    /**
+     * Counts the number of set bits (population counting) in a string stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @return Command Response - The number set bits in the string. Returns zero if the key is
+     *     missing as it treated as an empty string.
+     */
+    public T bitcount(@NonNull String key) {
+        ArgsArray commandArgs = buildArgs(key);
+        protobufTransaction.addCommands(buildCommand(Bitcount, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Counts the number of set bits (population counting) in a string stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @param start The starting offset.
+     * @param end The ending offset.
```

Comment:
```suggestion
     * @param start The starting byte offset.
     * @param end The ending byte offset.
```

Reply:
Thanks! Addressed in https://github.com/aws/glide-for-redis/commit/d6d52304d595ca9b35aed9e19024ffc9dc7772de